### DISCUSSION TITLE
Interface Delegation in Kotlin

### DISCUSSION THREAD

**Discussion by giscus** [PoisonedYouth/PoisonedYouth.github.io#10] [Announcements]
# Interface Delegation in Kotlin

Property delegation is not the only delegation.

https://poisonedyouth.github.io/Kotlin-Interface-Delegation

**Comment by SoftMoore** [comment]
Minor correction. In the first example, you show the output as follows:
`//Hello World!`
`//42`

But the output is actually:
`Printing String 'Hello World!'to console`
`Printing Int '42'to console`

**Reply by PoisonedYouth** [reply]
Thank you very much for the hint.

**Reply by SoftMoore** [reply]
One more minor correction to your correction.  You show the output as follows:
`//Printing String 'Hello World!'to console`
`//"Printing Int '42' to console"`
You should remove the double quotes from the second line of output.

Also, another minor suggestion, but not a correction.  I would add a space after the apostrophe (single quote) and the phrase "to console" in the first print line.  That is, change
`Printing String 'Hello World!'to console`
to
`Printing String 'Hello World!' to console`

But if you make this change, it should be made in both the source code and the output line.  What you have now for the output line is correct based on the source code.

Otherwise, great article.  Thanks.
